demo/iframe.html

```diff
@@ -5,75 +5,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Iframe</title>
     <script src="https://cdn.jsdelivr.net/npm/iframe-sync@latest/index.min.js"></script>
```

Comment:
```suggestion
    <script src="https://cdn.jsdelivr.net/npm/iframe-sync@1.0.4/index.min.js"></script>
```

I think we should start referring to the specific version of the script so that if people check out an old version, the demo from that version will work for them.